.husky/pre-commit

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+echo "Formating project and committing..."
+FILES=$(git diff --cached --name-only --diff-filter=ACMR | sed 's| |\\ |g')
+[ -z "$FILES" ] && exit 0
+
+# Prettify all selected files
+echo "$FILES" | xargs ./node_modules/.bin/prettier --ignore-unknown --ignore-path .gitignore --write --plugin-search-dir=.
```

Comment:
@melMass do you think we should limit this to .js, .jsx and maybe .scss files?

Reply:
It should be handled by prettier but we can restrict it before if needed